src/index.js

```diff
@@ -178,8 +178,7 @@ export default async function main(
   threshold,
   repository,
   verbose,
-  maxContributors,
-  maxLostContributors
+  branch = "master"
```

Comment:
shouldn't the default already be set from the cli param's defaults ?

Reply:
agree, I deleted the extra default